iteration/iteration_test.go

```diff
@@ -54,7 +53,7 @@ func (s *TestIterationRepository) TestCreateIteration() {
 		SpaceID:    fxt.Spaces[0].ID,
 		StartAt:    &start,
 		EndAt:      &end,
-		UserActive: &userActive,
+		UserActive: false,
```

Comment:
This line can be removed.

Reply:
@kwk will fix this

Reply:
working on this issue here -> https://github.com/fabric8-services/fabric8-wit/pull/1646